testkit/src/builder.rs

```diff
@@ -117,34 +114,20 @@ pub struct TestKitBuilder {
     our_validator_id: Option<ValidatorId>,
     test_network: Option<TestNetwork>,
     logger: bool,
-    rust_services: Vec<Box<dyn ServiceFactory>>,
-    runtimes: Vec<(u32, Box<dyn Runtime>)>,
+    rust_runtime: RustRuntime,
```

Comment:
Nit: Do we have to distinguish rust runtime on the field level?

Reply:
Yes. If we store it as a trait object like other runtimes we lose the ability to populate it with service factories.